css/style.css

```diff
@@ -200,3 +199,20 @@ a:link {
 #kingchooser {
   display: none;
 }
+
+.tooltip {
+  position: absolute;
+  transition: opacity 1s;
+  opacity: 1;
+  border: thin solid #333;
+  background-color: #161616;
+  border-radius: 5px;
+  padding: 1%;
+  color: #fff;
+  font-size: 2vw Arial;
```

Comment:
you can't put `Ariel` inside the `font-size` attribute. It needs to be `font`, or you have to move it to its own `font-family` specification.